backend/package.json

```diff
@@ -63,16 +65,16 @@
 		"@typescript-eslint/parser": "^2.8.0",
 		"chai": "^4.2.0",
 		"copy-webpack-plugin": "^5.0.5",
-		"eslint": "^6.6.0",
+		"lodash": "^4.17.15",
```

Comment:
Lodash is already a dependency (not a *dev* dependency)

Reply:
You are right. My mistake